feconf.py

```diff
@@ -608,13 +608,14 @@ def get_empty_ratings():
 ])
 
 # The header for the "Featured Activities" category in the library index page.
```

Comment:
I thought I had changed this. May I forgot to save before pushing it. Sorry for this! 
I will make this change.